build

```diff
@@ -71,7 +71,14 @@ make_relup() {
     local lib_dir="_build/$PROFILE/rel/emqx/lib"
     local releases_dir="_build/$PROFILE/rel/emqx/releases"
     local name_pattern
-    name_pattern="${PROFILE}-$(./scripts/pkg-full-vsn.sh 'vsn_matcher')"
+    # 4.5.0 upgraded OTP from 24.1.5-3 to 24.3.4.2-1, so we need to
+    # fool the matcher to make it pick up the previous 4.4.X releases
+    # with older OTP.
+    if [[ "${PKG_VSN}" == 4.5.0* ]]; then
```

Comment:
we should consider the cases when upgrading from 4.4.[8-9] to 4.5.[0-9] ?

Reply:
for 4.5.* versions, we need to use this new pattern to match all 4.5 base versions.
but in addition, we should also list (fine to hard code) all possible base versions (4.4.8 onward)

Reply:
The possible base versions are already enumerated in `relup-base-vsns.sh`.  I'll adapt that for more future 4.4+ and 4.5+ versions.